packages/react/src/core/printTheme.ts

```diff
@@ -1,50 +1,53 @@
 import { Theme } from './theme';
 
+const colorHexBlack = '#000000';
```

Comment:
Not sure we need these vars to be honest. Gzip will negate all of the repetition of `#000000` anyway and without the need for 2 more variables